salt/modules/boto_elb.py

```diff
@@ -115,6 +115,39 @@ def exists(name, region=None, key=None, keyid=None, profile=None):
         return False
 
 
+def get_all_elbs(region=None, key=None, keyid=None, profile=None):
+    '''
+    Return all load balancers associated with an account
+
+    CLI example:
+
+    .. code-block:: bash
+
+        salt myminion boto_elb.get_all_elbs region=us-east-1
+    '''
+    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+
+    try:
+        return [e for e in conn.get_all_load_balancers()]
+    except boto.exception.BotoServerError as error:
+        log.debug(error)
```

Comment:
Oh, heh....  Duh!

I cut-n-pasted from the exists() function above it, which does the same, but I agree this should be a bit more noisy for errors...  Fixing.